src/tests/test_browser_apis.py

```diff
@@ -224,6 +224,60 @@ def foo(obj):
     remove_event_listener(x, "click", foo)
 
 
+@run_in_pyodide
+async def test_additional_event_listener_arguments(selenium):
+    from pyodide.code import run_js
+
+    x = run_js(
+        """
+        class MockObject {
+            constructor() {
+                this.listeners = {};
+            }
+            addEventListener(event, handler, kwargs) {
+                if (event in this.listeners) {
+                    this.listeners[event].push(
+                        {
+                            handler: handler,
+                            kwargs: kwargs
+                        });
+                }
+                else {
+                    this.listeners[event] = [
+                        {
+                            handler: handler,
+                            kwargs: kwargs
+                        }
+                        ];
+                }
+            }
+            removeEventListener(event, handler, kwargs) {
+                if (event in this.listeners) {
+                    this.listeners[event] = this.listeners[event].filter(
+                        (existingHandler) => existingHandler.handler !== handler
+                    )
+                }
+            }
+        }
+        let x = new MockObject();
+        x;
+        """
+    )
+
+    from pyodide.ffi.wrappers import add_event_listener, remove_event_listener
+
+    def foo(obj):
+        pass
+
+    add_event_listener(x, "click", foo, once=True, something_else="foo")
+
+    assert x.listeners.click[0].kwargs.once is True
+    assert x.listeners.click[0].kwargs.something_else == "foo"
```

Comment:
Probably needs to be:
```suggestion
    assert x.listeners.click[0].kwargs["once"] is True
    assert x.listeners.click[0].kwargs["something_else"] == "foo"
```